### PR TITLE
Init model

### PR DESCRIPTION
**Pull request by huongnguyenmta**

### CODE REVIEW

File: test/fixtures/amanities.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (dogiadat) [inline test/fixtures/amanities.yml]: xoa het file khong dung di

**Comment by dogiadat** [comment]
ok

File: app/models/business_type.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class BusinessType < ApplicationRecord
+  has_many :business_types, dependent: :destroy
```
Comment (tiennguyenuet) [inline app/models/business_type.rb]: has many role

File: db/schema.rb [hunk 3]
```diff
@@ -10,7 +10,226 @@
...
+    t.integer  "location_rate"
+    t.datetime "created_at",    null: false
+    t.datetime "updated_at",    null: false
+  end
+
+  create_table "review_places", force: :cascade do |t|
+    t.integer  "user_id"
+    t.integer  "place_id"
+    t.string   "title"
+    t.string   "content"
```
Comment (cuongncFramgia) [inline db/schema.rb]: text

File: db/schema.rb [hunk 4]
```diff
@@ -10,7 +10,226 @@
...
+    t.string   "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "review_businesses", force: :cascade do |t|
+    t.integer  "user_id"
+    t.integer  "business_id"
+    t.string   "title"
+    t.string   "content"
```
Comment (cuongncFramgia) [inline db/schema.rb]: text

File: db/schema.rb [hunk 5]
```diff
@@ -10,7 +10,226 @@
...
+    t.integer  "user_id"
+    t.integer  "target_id"
+    t.integer  "type"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "notes", force: :cascade do |t|
+    t.string   "title"
+    t.string   "content"
```
Comment (cuongncFramgia) [inline db/schema.rb]: text

File: db/schema.rb [hunk 6]
```diff
@@ -10,7 +10,226 @@
...
+    t.string   "title"
+    t.string   "description"
+    t.datetime "created_at",  null: false
+    t.datetime "updated_at",  null: false
+  end
+
+  create_table "journey_places", force: :cascade do |t|
+    t.decimal  "latitude"
+    t.decimal  "longitude"
+    t.string   "description"
```
Comment (cuongncFramgia) [inline db/schema.rb]: text

File: db/schema.rb [hunk 7]
```diff
@@ -10,7 +10,226 @@
...
+    t.integer  "type"
+    t.string   "image"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "journey_diaries", force: :cascade do |t|
+    t.integer  "user_id"
+    t.string   "title"
+    t.string   "description"
```
Comment (cuongncFramgia) [inline db/schema.rb]: text

File: db/schema.rb [hunk 8]
```diff
@@ -10,7 +10,226 @@
...
+    t.time     "end_time"
+    t.string   "phone_number"
+    t.string   "hotel_class"
+    t.datetime "created_at",       null: false
+    t.datetime "updated_at",       null: false
+  end
+
+  create_table "comments", force: :cascade do |t|
+    t.integer  "user_id"
+    t.string   "content"
```
Comment (cuongncFramgia) [inline db/schema.rb]: text

File: db/schema.rb [hunk 9]
```diff
@@ -10,7 +10,226 @@
...
+  create_table "amanity_businesses", force: :cascade do |t|
+    t.integer  "business_id"
+    t.integer  "amanity_id"
+    t.datetime "created_at",  null: false
+    t.datetime "updated_at",  null: false
+  end
+
+  create_table "business_events", force: :cascade do |t|
+    t.string   "title"
+    t.string   "description"
```
Comment (cuongncFramgia) [inline db/schema.rb]: text

**Pull request merged**